### DISCUSSION TITLE
différent problème et message d'érreurs.

### DISCUSSION THREAD

**Discussion by ali13** [mastervanleeuwen/J-TrackGallery#65] [Q&A] [answered]
[-FR-]
Bonjour,
J'essais désespérément de créer une catégorie ou de renommé une catégorie existante. Mais j'obtiens un message d'érreurs.
que voici: **##Une erreur est survenue**
**1364** _Field 'ordering' doesn't have a default value_
Ma version de joomla est la 4.3.4
Je ne vois pas comment réglé ce problème.
MErci A+

[-EN-]
Hello,
I'm desperately trying to create a category or rename an existing category. However, I'm getting an error message.
Here it is: ##An error occurred
1364 Field 'ordering' doesn't have a default value
My version of Joomla is 4.3.4.
I don't see how to solve this problem.
Thanks, take care!

**Comment by mastervanleeuwen** [comment, marked as answer]
Hi,

Thanks for your message. This error was present in a previous version, but it should have been fixed in the mean time. Could you try installing the most recent pre-release: https://github.com/mastervanleeuwen/J-TrackGallery/releases/tag/0.9.34-rc39

and let me know whether that fixes the issue?

Best regards, Marco.

**Comment by ali13** [comment]
[- Fr -]
Bonjour,
J'ai installé la version mentionné plus haut.
exactement le pack package_jtg.zip.
Je n'est plus d'erreur en créant une catégorie
par contre j'ai toujours l'erreur en important une trace pour cette catégorie.
l'erreur est la suivante : 1364 Field 'icon_n' doesn't have a default value

à l'import de la trace il n'y a pas de champ icon, du coup je ne sais pas
Merci, en te souhaitant une bonne journée.

[- En -]
Hello,
I have installed the version mentioned above.
exactly the package_jtg.zip.
I get no more errors when creating a category
but I still get the error when importing a trace for this category.
the error is as follows: 1364 Field 'icon_n' doesn't have a default value

when importing the trace there is no icon field, so I don't know
Thank you and have a nice day.

[- Ajout -]
Je ne sais pas si ça peut aider, mais lorsque j'appel la fonction j'obtiens une erreur 500. Visible en mode debug.

I don't know if this helps, but when I call the function I get a 500 error. Visible in debug mode.

**Comment by mastervanleeuwen** [comment]
Thanks for the report. This was a bug that I overlooked in a previous update, and have now fixed in a new pre-release:

https://github.com/mastervanleeuwen/J-TrackGallery/releases/tag/0.9.34-rc40

Please try this version and let me know if the problem persists.

Best regards, Marco
